corehq/util/test_utils.py

```diff
@@ -0,0 +1,11 @@
+from functools import wraps
+from django.conf import settings
+
+
+def unit_testing_only(fn):
+    @wraps(fn)
+    def inner(*args, **kwargs):
+        assert settings.UNIT_TESTING, \
```

Comment:
we may want to do a real exception here since at some point we might turn off asserts on prod